docs/src/components/Code/Code.js

```diff
@@ -13,13 +13,13 @@ theme.plain.backgroundColor = undefined;
 theme.plain.fontFamily = undefined;
 
 const propTypes = {
-  children: pt.oneOfType([pt.element, pt.arrayOf(pt.element)]),
+  children: pt.oneOfType([pt.node, pt.arrayOf(pt.node)]),
   className: pt.string,
   live: pt.bool,
   title: pt.string,
 };
 
-const Code = ({ children, className, live = true, title, ...rest }) => {
+const Code = ({ children = [], className, live = true, title, ...rest }) => {
```

Comment:
I don't think we should be defining an empty array for children as a default. We should instead be checking anywhere we are trying to access children and make sure it is present.

Reply:
This was added to make `docs` work at some poinnt,  it will be removed in the next commit.